heron/ui/src/python/main.py

```diff
@@ -134,14 +136,15 @@ def main():
 
   # log additional information
   command_line_args = vars(parsed_args)
-  address = socket.gethostbyname(socket.gethostname())
+
+  address = '0.0.0.0' if command_line_args['address'] is None else command_line_args['address']
```

Comment:
instead of this - can we do the following - 
```
address = socker.gethostbyname(socket.gethostname()) if not command_line_args['address'] else command_line_args['address']
```